resources/metadata/kubernetesDeploy.yaml

```diff
@@ -223,6 +223,24 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+      - name: valuesMapping
+        type: "map[string]interface{}"
+        longDescription: |
+          Mapping of values provided by Piper onto custom paths in format `[custom-path]: [piper-value]`
+
+          Example:
+          ```yaml
+          valuesMapping:
+            subchart1.image.tag:        image.tag
```

Comment:
We decided against making `image.tag` available if there is more than one image.